api/backend.go

```diff
@@ -298,20 +304,6 @@ func (b *StatusBackend) registerHandlers() error {
 		return b.AccountManager().Accounts()
 	})
 
-	rpcClient.RegisterHandler(params.SendTransactionMethodName, func(ctx context.Context, rpcParams ...interface{}) (interface{}, error) {
```

Comment:
Why is this handler removed?

This will effectively reroute `eth_sendTransaction` requests from dApps to the default  implementation of `SendTransaction`  in go-ethereum, right? Then `(*StatusBackend) SendTransaction()` method becomes obsolete. I'm a bit confused here.

cc @mandrigin

Reply:
I was confused by this as well. I thought there were other calls to `backend.SendTransaction()` outside of the RPC one. In either case, if we were to keep this we'd need to pull re-register the handler using the password whenever an account is selected (so that it can call `backend.SendTransaction()`)

If we don't have this handler, where would status-react call SendTransaction from? @flexsurfer

Reply:
Okay, so this handler should be removed, along with PersonalSign handler.

Reply:
I gave it some thought and I think it should stay. Even if `eth_sendTransaction` should be intercepted on status-react side, what will happen if actually, DApp manages to call `eth_sendTransaction` or will be called via HTTP when playing with `statusd`? I guess we should create a handled that returns an error if `eth_sendTransaction` is called.